src/services/usersService.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const { UserModel } = require('../models/userModel.js');
+const { sequelize } = require('sequelize');
+
+const getAll = async() => {
+  const users = await UserModel.findAll({
+    order: [
+      'spentAt',
+    ],
+  });
+
+  return users;
+};
+
+const getById = async(userId) => {
+  const foundUser = await UserModel.findByPk(userId);
+
+  return foundUser;
+};
+
+const addUser = async(name) => {
+  const createdUser = await UserModel.create({ name });
+
+  return createdUser;
+};
+
+const removeUser = async(userId) => {
+  if (!getById(userId)) {
+    return false;
+  }
+
+  await UserModel.destroy({
+    where: {
+      id: userId,
+    },
+  });
+
+  return true;
+};
+
+const updateUser = async(userId, name) => {
+  if (!getById(userId)) {
+    return null;
+  }
+
+  const updatedUser = await UserModel.update({ name }, {
+    where: {
+      id: userId,
+    },
+  });
+
+  return updatedUser;
+};
+
+const reset = async() => {
+  const users = await getAll();
+
+  return sequelize.transaction(async(t) => {
+    for (const user of users) {
+      await removeUser(user.id);
+    }
+  });
```

Comment:
This can be improved, using transaction is good, but not needed as SQL databases have `TRUNCATE` command that removed all data from table in only one request. You can do it by `UserModel.destroy({truncate: true})` or `UserModel.truncate()`.
[Docs1](https://sequelize.org/api/v6/class/src/model.js~model#static-method-truncate), [docs2](https://sequelize.org/api/v6/class/src/model.js~model#static-method-destroy)